### PR TITLE
Replace heavy jquery ui with lightweight autocomplete library

### PR DESCRIPTION
**Pull request by iqit-commerce**
It is not neccesary to load so heavy library just for autocomplete. Ticket http://forge.prestashop.com/browse/BOOM-2696

### CODE REVIEW

File: ps_searchbar.js [hunk 1]
```diff
@@ -4,32 +4,29 @@ $(document).ready(function () {
...
             .then(function (resp) {
                 response(resp.products);
             })
             .fail(response);
         },
-        select: function (event, ui) {
-            var url = ui.item.url;
-            window.location.href = url;
+        renderItem: function (product, search) {
+            console.log(search);
```
Comment (kpodemski) [inline ps_searchbar.js]: oops

File: ps_searchbar.js [hunk 2]
```diff
@@ -4,32 +4,29 @@ $(document).ready(function () {
...
+            return '<div class="autocomplete-suggestion" data-url="' + product.url + '">' +
```
Comment (kpodemski) [inline ps_searchbar.js]: maybe as a proper function? to make it overridable

File: ps_searchbar.js [hunk 3]
```diff
@@ -4,32 +4,29 @@ $(document).ready(function () {
...
+            '<span class="category">' + product.category_name + '</span>' +
+            '<span class="separator"> > </span>' +
+            '<span class="product">' + product.name + '</span>' +
+            '</div>';
         },
+        onSelect: function (e, term, item) {
+            var url = item.data('url');
+            window.location.href = url;
```
Comment (kpodemski) [inline ps_searchbar.js]: not need for having url variable

File: ps_searchbar.js (hunk 1 above)
Comment (LittleBigDev) [inline ps_searchbar.js]: Yes, please remove the `console.log`

File: ps_searchbar.js (hunk 2 above)
Comment (LittleBigDev) [inline ps_searchbar.js]: Good idea.

File: ps_searchbar.js (hunk 3 above)
Comment (LittleBigDev) [inline ps_searchbar.js]: Agreed.

**Comment by mickaelandrieu** [comment]
@iqit-commerce what is the status of this pull request? Can you please address the comments if you have time?

If you can't please close it, it's ok :)

Have a nice day

File: ps_searchbar.js (hunk 1 above)
Comment (Progi1984) [reply ps_searchbar.js]: @iqit-commerce Could you remove it ?

File: ps_searchbar.js (hunk 2 above)
Comment (Progi1984) [reply ps_searchbar.js]: @iqit-commerce Could you use a proper function ?

File: ps_searchbar.js (hunk 3 above)
Comment (Progi1984) [reply ps_searchbar.js]: @iqit-commerce Please don't use a url variable.

File: jquery.auto-complete.css [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+}
+.autocomplete-suggestion {
+    position: relative;
+    padding: .3rem .6rem;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    border-top: 1px solid rgba(160, 160, 160, 0.25);
+}
+.autocomplete-suggestion:first-child {
+	border-top: none;
```
Comment (Progi1984) [inline jquery.auto-complete.css]:
```suggestion
   border-top: none;
```

**Comment by Progi1984** [comment]
Hello @iqit-commerce , thanks for your contribution, would you mind to rebase your pull request? Thanks

**Comment by matks** [comment]
Hi everyone,

It looks like this PR is not moving anymore. As mentioned in our [Maintainer Guide](https://devdocs.prestashop.com/1.7/project/maintainers-guide/reviewing-pull-requests/#stale-pull-requests), "Pull Requests may be closed after 30 days of inactivity following a request for modifications".

I close this pull request as it seems stale. If however you want to continue, we can re-open it 😉 . Please ping me if you come back, I will reopen the Pull Request. Thank you for your contribution and have a nice day.

**Pull request closed without merging**